src/components/atoms/EditorMonoPaneTitle/EditorMonoPaneTitle.tsx

```diff
@@ -0,0 +1,27 @@
+import {Typography} from 'antd';
+
+import styled from 'styled-components';
+
+import {FontColors} from '@styles/Colors';
+
+const {Text} = Typography;
+
+const EditorMonoPaneTitle = styled(Text)`
```

Comment:
Is this needed?

Reply:
The same reason as below comment created new to not affect all others.